src/OpenDiffix.Core/Value.fs

```diff
@@ -16,8 +16,8 @@ let rec toString value =
   | Boolean false -> "f"
   | Integer i -> i.ToString()
   | Real r -> r.ToString()
-  | String s -> String.quoteSingle s
-  | List values -> "[" + (values |> List.map toString |> String.joinWithComma) + "]"
+  | String s -> s
+  | List values -> "[" + (values |> List.map (toString >> String.quoteSingle) |> String.joinWithComma) + "]"
```

Comment:
We need to pass context data or call another function, otherwise this will quote everything, including numbers and other lists. A 2D list of numbers would print as `[ '[1,2]', '[3,4]' ]` which is not desired.

Reply:
OTOH this could work:

```fs
let rec toString value =
  match value with
  | Null -> "NULL"
  | Boolean true -> "t"
  | Boolean false -> "f"
  | Integer i -> i.ToString()
  | Real r -> r.ToString()
  | String s -> String.quoteSingle s
  | List values -> "[" + (values |> List.map toString |> String.joinWithComma) + "]"
  
  and toStringListItem value =
    match value with
    | String s -> String.quoteSingle s
    | value -> toString value
  ```

Reply:
yup, this makes sense, I think this check's all the boxes